apps/meteor/app/livechat/imports/server/rest/appearance.ts

```diff
@@ -52,8 +53,37 @@ API.v1.addRoute(
 				throw new Error('invalid-setting');
 			}
 
+			const dbSettings = await Settings.find({ _id: { $in: validSettingList } }, { projection: { _id: 1, value: 1, type: 1 } }).toArray();
```

Comment:
Avoid making queries from outside of the model, instead create a new function in the model that runs the query you need.